Concepts: perpetual markets (#765)
vegaprotocol/documentation: docs/concepts/trading-on-vega/settlement.md

---
sidebar_position: 7
title: Settlement
hide_title: false
description: The automated settlement process is facilitated by the protocol.
---

> Vega can be used to create cash-settled derivatives markets - futures as well as perpetual futures, meaning they are margined and settled in a single asset, as opposed to being settled in the actual (physical) underlying asset.

Reviewer: ```suggestion
Vega can be used to create cash-settled derivatives markets - futures as well as perpetual futures - meaning they are margined and settled in a single asset, as opposed to being settled in the actual (physical) underlying asset.
```

:::note Read more
[Mark to market](./positions-margin.md#mark-to-market)
:::

## Periodic settlement for perpetuals 
Perpetual futures markets are settled periodically using spot prices from an external data source. This is in addition to mark to market settlement, which happens when the mark price changes. 

These periodic settlements are known as the funding periods.

The data source for spot prices and how often the market is scheduled to use that data to settle positions are defined in the market's parameters.

When a settlement schedule time is reached, the protocol calculates how much should be transferred between the parties for each open position. This is based on the provided price data for the underlying asset. The difference between the market's mark price and the spot price of the underlying asset is used to determine how much is moved between the two sides of each open position.

> If the perpetual price is above the spot price, the funding rate is positive and those with long positions "pay" those who are short in a position.  If the perpetual price is below the spot price, the funding rate is negative and traders with short positions lose money while those with long positions gain.

Reviewer: ```suggestion
If the perpetual price is above the spot price, the funding payment is positive and those with long positions "pay" those who are short in a position.  If the perpetual price is below the spot price, the funding payment is negative and traders with short positions lose money while those with long positions gain.
```

The amount depends on how far apart the spot price is from the perpetual price on the Vega market.

> When executing the settlement for each funding period, first cashflows are collected from the parties making the payment, and then those assets are distributed to those receiving it. Any profit or loss as the result of the funding rate is only final once the position is closed, and is tracked as unrealised P&L.

Reviewer: ```suggestion
When executing the settlement for each funding period, first cashflows are collected from the parties making the payment, and then those funds are distributed to those receiving it. Any profit or loss as the result of the funding rate is only final once the position is closed, and is tracked as unrealised P&L until then.
```
